homework_4/bet_acceptance_api/cmd/bootstrap/api.go

```diff
@@ -0,0 +1,43 @@
+package bootstrap
+
+import (
+	"code-cadets-2021/homework_4/bet_acceptance_api/cmd/config"
+	"code-cadets-2021/homework_4/bet_acceptance_api/internal/api"
+	"code-cadets-2021/homework_4/bet_acceptance_api/internal/api/controllers"
+	"code-cadets-2021/homework_4/bet_acceptance_api/internal/api/controllers/validators"
+	"code-cadets-2021/homework_4/bet_acceptance_api/internal/domain/services"
+	"code-cadets-2021/homework_4/bet_acceptance_api/internal/infrastructure/rabbitmq"
+	"github.com/streadway/amqp"
+)
+
+func newBetAcceptanceValidator(coeffUpperBound, paymentLowerBound, paymentUpperBound float64) *validators.BetAcceptanceValidator {
+	return validators.NewBetAcceptanceValidator(coeffUpperBound, paymentLowerBound, paymentUpperBound)
+}
+
+func newBetReceivedPublisher(publisher rabbitmq.QueuePublisher) *rabbitmq.BetReceivedPublisher {
+	return rabbitmq.NewEventUpdatePublisher(
+		config.Cfg.Rabbit.PublisherExchange,
+		config.Cfg.Rabbit.PublisherBetReceivedQueue,
+		config.Cfg.Rabbit.PublisherMandatory,
+		config.Cfg.Rabbit.PublisherImmediate,
+		publisher,
+	)
+}
+
+func newBetAcceptanceService(publisher services.BetReceivedPublisher) *services.BetAcceptanceService {
+	return services.NewBetAcceptanceService(publisher)
+}
+
+func newController(betAcceptanceValidator controllers.BetAcceptanceValidator, betAcceptanceService controllers.BetAcceptanceService) *controllers.Controller {
+	return controllers.NewController(betAcceptanceValidator, betAcceptanceService)
+}
+
+// Api bootstraps the http server.
+func Api(rabbitMqChannel *amqp.Channel) *api.WebServer {
+	betAcceptanceValidator := newBetAcceptanceValidator(10.0, 2.0, 100.0)
```

Comment:
This could probably be set in a configuration file. You don't want to change your bootstrap code to change a value like payment boundaries.